### DISCUSSION TITLE
Progress bar transparent background

### DISCUSSION THREAD

**Discussion by Kens-Jaswal** [saadeghi/daisyui#1290] [Q&A] [answered]
how can I transparent progress bar background and show only moving tracker on it?

pls help

**Comment by saadeghi** [comment, marked as answer]
Here's an example: https://play.tailwindcss.com/hcNDbVOeDM
```html
<progress class="progress w-56 bg-transparent [&::-webkit-progress-bar]:bg-transparent" value="40" max="100">
</progress>
```

We should use both `bg-transparent` and `[&::-webkit-progress-bar]:bg-transparent`  
Because the background color of a `<progress>` element is defined by the element itself on Firefox but it's defined by `::-webkit-progress-bar` on Chrome, Safari, etc.

**Reply by esphoenixc** [reply]
Sir, how can I change the moving bar's color? how can I target the moving bar's color to change its color?

**Comment by Kens-Jaswal** [comment]
Thanks for your reply...it works well..
